guvnor-structure/guvnor-structure-client/src/main/java/org/guvnor/structure/client/advnavigator/TreeNavigator.java

```diff
@@ -111,38 +111,38 @@ public void callback( final NavigatorContent response ) {
 
     @Override
     public void loadContent( final Path path ) {
-        final NavigatorItem parent = new TreeNavigatorItemImpl( new TreeItem( TreeItem.Type.FOLDER, path.getFileName() ) );
+        final NavigatorItem parent = new TreeNavigatorItemImpl( new FSTreeItem( FSTreeItem.FSType.FOLDER, path.getFileName() ) );
         tree.addItem( ( (TreeNavigatorItemImpl) parent ).parent );
 
         loadContent( parent, path );
     }
 
-    private boolean needsLoading( final TreeItem item ) {
+    private boolean needsLoading( final FSTreeItem item ) {
         return item.getChildCount() == 1 && LAZY_LOAD.equals( item.getChild( 0 ).getText() );
     }
 
     private class TreeNavigatorItemImpl implements NavigatorItem {
 
-        private final TreeItem parent;
+        private final FSTreeItem parent;
 
-        TreeNavigatorItemImpl( final TreeItem treeItem ) {
+        TreeNavigatorItemImpl( final FSTreeItem treeItem ) {
             this.parent = checkNotNull( "parent", treeItem );
         }
 
         public void addDirectory( final Path child ) {
             checkCleanupLoading();
 
             //Util.getHeaderSafeHtml( images.openedFolder(), child.getFileName() )
-            final TreeItem newDirectory = parent.addItem( TreeItem.Type.FOLDER, child.getFileName() );
-            newDirectory.addItem( TreeItem.Type.LOADING, LAZY_LOAD );
+            final FSTreeItem newDirectory = parent.addItem( FSTreeItem.FSType.FOLDER, child.getFileName() );
+            newDirectory.addItem( FSTreeItem.FSType.LOADING, LAZY_LOAD );
             newDirectory.setUserObject( child );
         }
 
         public void addFile( final Path child ) {
             checkCleanupLoading();
 
             //Util.getHeaderSafeHtml( images.file(), child.getFileName() )
```

Comment:
@alepintus @romartin when you guys have a chance, delete dead comments like these.